nir/src/main/scala/scala/scalanative/nir/Arg.scala

```diff
@@ -0,0 +1,40 @@
+package scala.scalanative.nir
+
+/**
+ * Created by Kamil on 23.07.2016.
+ */
```

Comment:
We generally don't use authorship comments in Scala Native. `git blame` is quite good at it anyway.